response/slack/models/comms_channel.py

```diff
@@ -65,7 +67,17 @@ def post_in_channel(self, message: str):
         settings.SLACK_CLIENT.send_message(self.channel_id, message)
 
     def rename(self, new_name):
-        settings.SLACK_CLIENT.rename_channel(self.channel_id, new_name)
+        if new_name:
+            try:
+                response = settings.SLACK_CLIENT.rename_channel(
+                    self.channel_id, new_name
+                )
+                self.channel_name = response["channel"]["name"]
+                self.save()
+            except SlackError as e:
+                logger.error(
+                    f"Failed to rename channel {self.channel_id} to {new_name}. Error: {e}"
+                )
```

Comment:
nit : Should we log or raise and error if new_name is empty ? Should we also look at the size of the new_name